WooCommerce/Classes/ViewRelated/Dashboard/Factories/StatsDataTextFormatter.swift

```diff
@@ -130,7 +132,7 @@ struct StatsDataTextFormatter {
 
     // MARK: Conversion Stats
 
-    /// Creates the text to display for the conversion rate.
+    /// Creates the text to display for the conversion rate based on SiteVisitStats data and a given interval.
     ///
     static func createConversionRateText(orderStats: OrderStatsV4?, siteStats: SiteVisitStats?, selectedIntervalIndex: Int?) -> String {
```

Comment:
Same suggestion as above
```suggestion
    static func createConversionRateText(orderStats: OrderStatsV4?, siteStats: SiteVisitStats?, selectedIntervalIndex: Int) -> String {
```